packages/Menu/src/index.ts

```diff
@@ -0,0 +1,6 @@
+import Menu from './Menu';
+import MenuGroup from './MenuGroup';
+import MenuItem from './MenuItem';
+
+export { MenuGroup, MenuItem };
+export default Menu;
```

Comment:
Do we ever think someone is going to use `Menu` without `MenuGroup` and `MenuItem`? If no, then I think we should keep them all as named exports.

Reply:
Think you're right, I don't think the default use-case would be to use Menu without MenuGroup/MenuItem. Fixed this.

Reply:
I would disagree – I think that it's likely there will be cases where someone may use Menu as a standalone component. MenuGroup and MenuItem are both included as easy and convenient ways to construct the content of Menu, but Menu can certainly be used standalone.

I'm thinking of it as a one to many relationship, where the styles of MenuGroup and MenuItem are specific to Menu, but not the other way around.

Reply:
My main thinking around a default export is that it's something that is usually used by itself, with named exports being ancillary things that are optional to use.

If we want all things exported to be used then they should all be named exports IMO. 

If we want to encourage that Menu be used with MenuItem then they should probably be all named exports.

If we think people will likely just use Menu by itself then having it as the default export makes sense.

The bigger question is, what does it mean to make something a default export?